src/ParseServer.js

```diff
@@ -361,7 +359,20 @@ function injectDefaults(options: ParseServerOptions) {
       ])
     );
 
-    options.protectedFields = { _User: { '*': userSensitiveFields } };
+    if (!('protectedFields' in options)) {
+      options.protectedFields = [];
+    }
+
+    if (!('_User' in options.protectedFields)) {
+      options.protectedFields['_User'] = [];
```

Comment:
Same here `={}`

Reply:
but this is covered.  it probably just gets overwritten.

Reply:
i'm wrestling with some oddities in the unit tests that if i can figure out will help me test this.